.circleci/main/commands/executions/run-integration-tests.yml

```diff
@@ -35,8 +35,8 @@ commands:
           when: always
           command: |
             mkdir -p ~/test-results/junit
-            find . -type f -regex ".*/target/.*-reports-[0-9]+/.*xml" -exec cp {} ~/test-results/junit/ \;
-            find . -type f -regex ".*/target/.*-reports-[0-9]+/.*dump.*" -exec cp {} ~/test-results/junit/ \;
+            find . -type f -regex ".*/target/surefire-reports/.*xml" -exec cp {} ~/test-results/junit/ \;
+            find . -type f -regex ".*/target/failsafe-reports/.*xml" -exec cp {} ~/test-results/junit/ \;
```

Comment:
You lost the dump files which sometimes are generated.

You could probably fix this by just getting rid of the `-[0-9]+` and leaving it otherwise alone.

Reply:
Ah, thank you. Do we possibly generate dump files with the smoke tests? If so, I’ll make all of them the same to grab xml and dump files.

Reply:
@RangerRick I made the change you requested. I also made the `find` commands in `smoke-test` the same as what is now in the integration tests so we'll get dump files from smoke tests, as well.